airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumPropertiesManager.kt

```diff
@@ -11,7 +11,8 @@ import java.util.*
 abstract class DebeziumPropertiesManager(
     private val properties: Properties,
     private val config: JsonNode,
-    private val catalog: ConfiguredAirbyteCatalog
+    private val catalog: ConfiguredAirbyteCatalog,
+    private val streamNames: List<String>
```

Comment:
Does it make sense to have a default value for the parameters so that we don't need to repetitively pass the empty list to the constructor? e.g.

```
private val catalog: ConfiguredAirbyteCatalog,
private val streamNames: List<String> = new ArrayList<String>()
```

Reply:
I'd rather keep it explicit, so that in the read phase don't actually set an empty list of streamnames. 

This will cause other certified connectors to fail compilation when using the new cdk, but I will send a quick follow up for that